README.md

```diff
@@ -12,22 +12,65 @@ by the generated snapshot flakes. However, the `flake.lock` in this repo
 is critically important, as it controls which versions of our upstream
 dependencies (nixpkgs, nix-ros-overlay) we are pinned to.
 
-## Usage
+## Nix Installation
```

Comment:
I'd personally shy away from this. This will always go stale, I'd link to the upstream documentation only.

Reply:
I'm of two minds. Yes, generally I'd agree, but given that this repo is specifically targeting nix novices, I don't want to send them on a wild goose chase of external links and wrangling configuration.